spec/card_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+require 'card'
+
+describe Card do
+  it 'intitializes with a suit and rank' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.